### DISCUSSION TITLE
Led Lighting!

### DISCUSSION THREAD

**Discussion by garyzz** [synman/Octoprint-Bettergrblsupport#97] [Ideas]
It would be inspiring to enable this plugin to support control of LED lighting on CNC. Neopixels coupled to Raspi Pi or any other lighting option where the plugin _running on a Raspi Pi_ could set color and brightness levels to make our  CNC machines dance more like eye candy.

**Comment by synman** [comment]
That's the great thing about Octoprint.... There are hundreds of plugins, and many of them work just fine alongside BGS.

Checkout the plugin repo for LED related plugins.

There are a 100 different ways to implement lighting already.  

On my 3d printer I use a shell script integrated with a couple smart plugs to control printer and lighting power with the PSU Control plugin, for example.

I also use PSU Control on my grbl machines without issue.
